types/board_support_common.yaml

```diff
@@ -0,0 +1,54 @@
+description: EV and EVSE board support types
+types:
+  BspEvent:
+    description: Event stream from ControlPilot signal/Relais
+    type: object
+    additionalProperties: false
+    required:
+      - event
+    properties:
+      event:
+        type: string
+        enum:
+          - 'A'
+          - 'B'
+          - 'C'
+          - 'D'
+          - 'E'
+          - 'F'
+          - 'PowerOn'
+          - 'PowerOff'
+          - 'ErrorDF'
+          - 'ErrorRelais'
+          - 'ErrorVentilationNotAvailable'
+          - 'ErrorOverCurrent'
+          - 'ErrorOverVoltage'
+          - 'ErrorUnderVoltage'
+          - 'ErrorMotorLock'
+          - 'ErrorOverTemperature'
+          - 'ErrorBrownOut'
+          - 'ErrorCablePP'
+          - 'ErrorEnergyManagement'
+          - 'ErrorNeutralPEN'
+          - 'ErrorCpDriver'
+          - 'PermanentFault'
+          - 'EvseReplugStarted'
+          - 'EvseReplugFinished'
+          - 'Disconnected'
```

Comment:
What is the difference between 'Disconnected' and 'A'?

Reply:
any updates?

Reply:
Disconnected is only used on the EV side. In principle the EV will measure 0V when it is not connected to a charging station but there is a logical difference between e.g. the car setting E to indicate a fault and the car detecting it is not connected at all. This may be changed later on when the EV branch is being merged, but for now we need it for the EV sim. I'll clarify in the documentation that this should be ignored for EVSE side.